xdist/dsession.py

```diff
@@ -1,4 +1,5 @@
 import difflib
+import importlib
```

Comment:
Thanks, I guess I can use `__import__` for that case. I thought that `pytest` has general workaround for that case, but as I can see it has only specialised methods like `import_plugin`.